### ISSUE TITLE
Training loss and validation loss negative

### ISSUE THREAD

**Issue by ioannispol** [ioannispol/UnderWaterU-Net#21]
Check the dataset to resolve the issue
